cmd/kogito/deploy_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func Test_DeployCmd_WhenThereAreNoOperator(t *testing.T) {
+	cli := fmt.Sprintf("deploy example-drools https://github.com/kiegroup/kogito-examples --context drools-quarkus-example --app kogito")
+	lines, _, err := executeCli(cli, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "kogito"}})
+	assert.Error(t, err)
+	assert.Contains(t, lines, "Couldn't find the Kogito Operator")
+}
+
+func Test_DeployCmd_WhenThereAreAnOperator(t *testing.T) {
```

Comment:
This test checks the situation that CRD exists in the cluster. But what about the case when Kogito is available in cluster, but not installed in namespace?

Reply:
We don't have an API for that yet. I'll add it as soon as we get it to the OperatorHub.

Reply:
ok, thanks